import/controls/permissionmodel.cpp

```diff
@@ -138,6 +143,7 @@ void PermissionModel::setPermissionList(const QVariantList &data)
                                            PermissionManager::intToExpiration(varMap.value("expireType").toInt())));
     }
     endResetModel();
+    emit countChanged();
```

Comment:
Not needed now but this should handle gracefully insert, add, and data changes. Filed bug 52375 about it.